Content/Library/src/MyLib.1/paket.references

```diff
@@ -1,2 +1,3 @@
 FSharp.Core
 Microsoft.SourceLink.GitHub
+Newtonsoft.Json
```

Comment:
Need to remove this before merge.  It was here to prove we could load 3rd party types in the docs